### PR TITLE
Toast & grid

### PR DESCRIPTION
**Pull request by amartya-rs**
feature update: toast & simplified grid

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code is well understandable. Good to merge

File: pages/docs.html [hunk 1]
```diff
@@ -499,6 +499,61 @@ <h5>SIGNUP</h5>
...
+
+    <div class="grid-container">
+        <div class="grid-3-column">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+        </div>
+    </div>
+
+    <!-----simplified-grid-end----->
```
Comment (iRohitGaur) [inline pages/docs.html:555]: clean and understandable. Good that you have placed comments to segregate sections

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Review by archit7448 — APPROVED** [review]
good to merge looks good

File: pages/docs.html (hunk 1 above)
Comment (amartya-rs) [reply pages/docs.html:555]: Thanks for your feedback.

**Pull request merged**